packages/demand-bloc-trip-create/src/AutocompleteBloc.ts

```diff
@@ -62,17 +62,14 @@ export class AutocompleteBloc implements TripCreateFieldItem {
   }
 
   private getFeatchedResults() {
-    const { minLengthToSearch, autocompleteDebounceTime } = this.options
-    const isValidLength = (text: string) => text.length > minLengthToSearch
+    const { minLengthToSearch = 2, autocompleteDebounceTime } = this.options
```

Comment:
Default value for `autocompleteDebounceTime` as well as for `minLengthToSearch`?